### ISSUE TITLE
Pages visible, but the communication site is still the default blank template

### ISSUE THREAD

**Issue by bn641** [pnp/custom-learning-office-365#158]
I setup the app catalog and manually uploaded v3.0.1 to the catalog in SPO. I was able to run the script to complete the provisioning by modifying a line to work with MFA "Connect-PnPOnline -Url $clSite -SPOManagementShell".

The CustomLearningViewer and CustomLearningAdmin pages are visible and load as well. My issue is that the home page of the communication site isn't customized like all of the screenshots I've seen. I also don't see links in the navigation for Microsoft 365 training and asking a question. Are those all custom configurations and everyone is just using those screenshots as examples?

**Comment by juliemturner** [comment]
That's because you only get those custom pages when you provision learning pathways using the provisioning service, not when you do a manual provision.

**Comment by bn641** [comment]
Thanks Julie, I'll try to provision another site using that. I had already tried to use the provisioning service a bunch of times, but it kept failing. 

I was hoping to avoid doing all of the customization on my own, since I like the look of the templates and want to use that as a baseline.

**Comment by juliemturner** [comment]
I don't know if the provisioning service supports MFA. Further, you can only have one learning pathways administrative site in a tenant, so if you provision learning pathways to a different site you will end up abandoning your original site. Further, I'm not sure how nice the provisioning service plays when you already have the app in the app catalog that in this case would be an older version.  The new version only addresses upgrade scenarios from v2 so it doesn't really matter which version you have but you should be aware.

**Comment by bn641** [comment]
I just retried the provisioning and it worked this time! I didn't get rid of my manual site beforehand, since I had started provisioning before I saw your comment. The only difference as far as I know is that when trying Connect-PnPOnline -PnPO365ManagementShell with the manual script it asked me to accept additional permissions to our tenant prior to failing.

Either way I'll close this out as it worked fine with the manual process per your comments.

Thanks again
